index.html

```diff
@@ -181,25 +194,24 @@ <h3>Skills</h3>
        </section>
     </section>
 
-    <footer class="contact">
+    <footer class="contact" id="contact">
         <section class="footer-text">
             <h2 class="contact-header">Contact me</h2>
             <p class="contact-text">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
         </section>
         <section>
-            <form class="contact-form" action="https://formspree.io/f/mleaokev" method="POST">
-                <input class="contact-input" type="text" name="name-input" placeholder="Enter name here" maxlength="30" required><br>
-                <input class="contact-input" type="email" placeholder="Enter email address here" name="email-input" required><br>
+            <form class="contact-form" id="contact-form" action="https://formspree.io/f/mleaokev" method="POST">
+                <input class="contact-input" id="contact-name" type="text" name="name-input" placeholder="Enter name here" maxlength="30" required><br>
+                <input class="contact-input" id="contact-email" type="email" placeholder="Enter email address here" name="email-input" required><br>
                 <textarea class="contact-input" name="textarea-input" maxlength="500" placeholder="write your message here" cols="30" rows="10" required></textarea><br>
-                <button class="btn3 rty btn1">
-                     <span class="btn-text">Get it Touch</span>
-                </button>
+                <small class="inter-font" id="email-error"></small><br>
```

Comment:
- [ ] You're missing the JavaScript linter. Modify the linters.yml file. Check [this link](https://github.com/microverseinc/linters-config/tree/master/html-css-js)

- [ ] I think the form will look better if you add some CSS to the error message or modify it to make it fit in one line.
![image](https://user-images.githubusercontent.com/43698511/132754347-362c58b2-02e0-44f6-bb83-663c65d052c7.png)
![image](https://user-images.githubusercontent.com/43698511/132754428-0dc07c3f-ab9e-4967-b5fd-db7021926183.png)